docs/NetCore_GettingStarted.md

```diff
@@ -25,29 +25,34 @@
 	- Do not separate analyzer tests from code fix tests. If the analyzer has a code fix, then write all your tests as code fix tests.
 		- Calling `VerifyCodeFixAsync(source, source)` verifies that the analyzer either does not produce diagnostics, or produces diagnostics where no code fix is offered.
 		- Calling `VerifyCodeFixAsync(source, fixedSource)` verifies the diagnostics (analyzer testing) and verifies that the code fix on source produces the expected output.
-- Run the analyzer locally against `dotnet/runtime` [instructions](#Testing-against-the-Runtime-repo). 
-	- Use the failures to discover nuance and guide the implementation details. 
-	- Run the analyzer against `dotnet/roslyn` [instruction](#Testing-against-the-Roslyn-repo), and with `dotnet/aspnetcore` if feasable. 
-	- Review each of the failures in those repositories and determine the course of action for each. 
+- Run the analyzer locally against `dotnet/runtime` and `dotnet/roslyn-analyzers` [instructions](#Testing-against-the-Runtime-and-Roslyn-analyzers-repo).
```

Comment:
As the steps for testing roslyn-analyzers similar to testing with runtime put them together and using same instruction